swift/vapor-framework/main.swift

```diff
@@ -0,0 +1,18 @@
+import Vapor
+
+let app = Application()
+defer { app.shutdown() }
+
+app.get { _ in
+    Response()
+}
+
+app.get("user", ":userID") { req in
+    req.parameters.get("userID") ?? ""
+}
+
+app.post("empty") { _ in
```

Comment:
it should be `app.post("user")` to handle `POST:/user`

Reply:
Ah apologies